libraries/azure_backend.rb

```diff
@@ -313,14 +313,14 @@ def get_api_version(provider, resource_type, api_version_status = 'latest')
       provider_details = @azure.provider_details[provider.to_sym]
     end
 
-    resource_type_details = provider_details[:resourceTypes].select { |rt| rt[:resourceType] == resource_type }&.first
+    resource_type_details = provider_details[:resourceTypes].select { |rt| rt[:resourceType].downcase == resource_type.downcase }&.first
```

Comment:
Slight nitpick, but I think in most cases we usually go with `upcase` for comparison

Reply:
Will change the other comparisons as well.